### DISCUSSION TITLE
Calendar ValueFormat Feature

### DISCUSSION THREAD

**Discussion by felixhoi** [primefaces/community#472] [PrimeVue]
Calendar always return Date object, is it possible to support a valueUseFormat boolean props , use like
```html
<Calendar v-bind="date" dateFormat="dd/mm/yy" valueUseFormat />
```
when valueUseFormat is given, return a string with specified dateFormat, otherwise return date object.
Thank you for consider the enhancement suggest.

**Comment by felixhoi** [comment]
if agree, I can try to make a pull request. thank you.

**Comment by a982246809** [comment]
i need this feture
